common/flatpak-context-private.h

```diff
@@ -76,6 +76,8 @@ struct FlatpakContext
   GHashTable            *session_bus_policy;
   GHashTable            *system_bus_policy;
   GHashTable            *generic_policy;
+  gboolean               fs_host_flags;
+  gboolean               fs_home_flags;
```

Comment:
These should of course have been `FlatpakFilesystemFlags` in this version of the branch; the boolean type was left over from them being a separate boolean `fs_host_reset`. But I think I've convinced myself that booleans would be better here, so I'll change that in the next revision of this branch.